modules/host/default.nix

```diff
@@ -4,18 +4,41 @@
   self,
   microvm,
   netvm,
-}: {lib, ...}: {
-  imports = [
-    (import ./minimal.nix)
+}: {
+  lib,
+  config,
+  pkgs,
+  ...
+} @ args: let
+  cfg = config.ghaf.host.minification;
+in
+  with lib; {
+    imports = [microvm.nixosModules.host];
 
-    microvm.nixosModules.host
+    options.ghaf.host.minification = with lib; {
+      reduceProfile = mkEnableOption "reduced profile";
```

Comment:
Here we could describe the reduced profile a bit more vocally - e.g. "Profile to reduce host size with NixOS supported options - see modules/host/minimal.nix and https://nixos.org/manual/nixos/stable/options.html for details"